### PR TITLE
[PLAYER-4099] Fix UI for iPads (Analytics)

### PR DESCRIPTION
**Pull request by cacel**
[PLAYER-4099] [iOS] Save analytics data when device is offline
Shows the log of the files saved in the device for Analytics offline.
QA reported: "On iPADs, The analytics window is overlapped on player". This PR solves this issue.

### CODE REVIEW

File: PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp/Info.plist [hunk 1]
```diff
@@ -36,24 +45,10 @@
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
-		<string>UIInterfaceOrientationLandscapeLeft</string>
```
Comment (orobles-ooyala) [inline PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp/Info.plist]: I'm not sure if this is correct approach. Does this affect the video fullscreen in landscape mode? Can't this be moved only to the view that you need and not the whole app?

**Review by maxkupetskii — CHANGES_REQUESTED** [review]
I can see that according to the ticket it requires another issue to be solved
_Save analytics data when device is offline_
Why all of a sudden it's a fix for UI?

**Comment by DmitriiMaslov** [comment]
Guys, I would also ask to provide clear description for PRs and if you see this is not done in other PRs please do the same.

**Comment by cacel** [comment]
@maxkupetskii The issue was solved in the iOS-SDK (core).
QA reported that the sample app was working fine for iPhone but for iPad the textView was overlapping the app. In the ticket you can see the last comment for QA: "On iPADs, The analytics window is overlapped on player."

@DmitriiMaslov fyi

**Comment by cacel** [comment]
@orobles-ooyala Now the app can be used landscape and portrait, I added the constants in the plist file for your previous comment.

**Review by orobles-ooyala — APPROVED** [review]

**Review by brol1dev — APPROVED** [review]
@DmitriiMaslov, @maxkupetskii  I don't want to merge this one until you do a final look. Check it out and if you feel ok you can merge it.

Thanks

**Pull request merged**